### PR TITLE
Newsletter 13: Add blitz-path

### PR DESCRIPTION
**Pull request by BezPowell**
Part of #232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -66,6 +66,11 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [blitz-path](https://github.com/BezPowell/blitz-path)
+[blitz-path](https://github.com/BezPowell/blitz-path) is a new crate providing an implementation of the [JPS](https://en.wikipedia.org/wiki/Jump_point_search) pathfinding algorithm.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:70]:
```suggestion
[blitz-path](https://github.com/BezPowell/blitz-path) is a new crate providing
an implementation of the [JPS](https://en.wikipedia.org/wiki/Jump_point_search)
pathfinding algorithm.
```

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+JPS is an optimization of the A* search algorithm for uniform-cost grids, which are common in games. While fully functional, the code is still in an early state and any suggestions for improvements - especially on how best to integrate it with the existing ecosystem - are greatly appreciated.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:72]:
```suggestion
JPS is an optimization of the A* search algorithm for uniform-cost grids, which
are common in games. While fully functional, the code is still in an early
state and any suggestions for improvements - especially on how best to
integrate it with the existing ecosystem - are greatly appreciated.
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good, just needs breaking onto more lines to satisfy the formatting CI checks:

**Comment by BezPowell** [comment]
> Content looks good, just needs breaking onto more lines to satisfy the formatting CI checks:

Thanks for the suggestions, I have applied them now. Does the url of links get taken into account for the line-length?

**Comment by 17cupsofcoffee** [comment]
They shouldn't do, I think it was a false positive because of there being trailing whitespace after the link.

File: content/posts/newsletter-013/index.md [hunk 3]
```diff
@@ -66,6 +66,16 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [blitz-path](https://github.com/BezPowell/blitz-path)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:69]:
```suggestion
### [blitz-path](https://github.com/BezPowell/blitz-path)

```

Comment (17cupsofcoffee) [reply content/posts/newsletter-013/index.md:69]: Linter is now complaining that there needs to be an extra linebreak here (why didn't it say that the first time 🤔)

**Comment by BezPowell** [comment]
I'm terribly sorry, I seem to be making a lot of unnecessary work for you. I ran it through a linter provided by a VSCode extension originally and as it didn't complain assumed it was ok, it looks like it actually didn't report any errors because it wasn't working :laughing:

**Comment by 17cupsofcoffee** [comment]
It's no problem at all, I don't know of an easy way of running the lint config locally so I also make mistakes in my own PRs 😅

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Thanks for the PR!

> It's no problem at all, I don't know of an easy way of running the lint config locally so I also make mistakes in my own PRs

@17cupsofcoffee I'm using https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint vscode extension to auto-run all the lints with our config file locally. I guess this (along with a style/formatting guide) should be documented in CONTRIBUTING.md or something.. :thinking:

**Pull request merged**